mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -136,6 +152,59 @@ AUI.add(
 							taskInput.val('');
 
 							instance._updateTaskCounts();
+
+							var taskForm = instance.byId('taskForm');
+
+							var resourceURL = taskForm.attr('action');
+
+							var taskInputValue = A.one('#<portlet:namespace />task-input');
+
+							if (resourceURL) {
+								A.io.request(
+									resourceURL,
+									{
+										method: 'post',
+
+										data: {
+											taskInputValue: taskInputValue
+										},
+
+										form: {
+											id: taskForm
+										},
+
+										on: {
+											failure: function() {
+												alert(this.get('responseData'));
```

Comment:
@mateomustapic I have feeling you forgot to remove debugging statement? If this has purpose, please forget my comment